commons/src/main/kotlin/com/simplemobiletools/commons/activities/TestManageBlockedNumbers.kt

```diff
@@ -105,6 +106,9 @@ class TestManageBlockedNumbers : BaseSimpleActivity() {
                     },
                     onCopy = { blockedNumber ->
                         copyToClipboard(blockedNumber.number)
+                    },
+                    canScroll = { canPerformScroll ->
+                        canScroll = canPerformScroll
```

Comment:
isnt this a bit too complicated? We should always assume that this screen can be scrolled, even if no contacts are blocked

Reply:
yeah, it's now a PoC, I don't like the code either, will come up with a better solution, this was just to see if it's working as intended.